repos/system_upgrade/el8toel9/actors/opensslconfigcheck/library/opensslconfigcheck.py

```diff
@@ -0,0 +1,58 @@
+from leapp.libraries.common import api
```

Comment:
@Jakuje Please rename the directory to "libraries" instead of "library". Thanks...